Packs/XSOARFileManagement/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "XSOAR File Management",
     "description": "This pack let user manipulate file inside XSOAR more easily than with the builtin functions.",
     "support": "community",
-    "currentVersion": "1.1.1",
+    "currentVersion": "1.2.0",
```

it's a minor change, and change the name of rn file as well
```suggestion
    "currentVersion": "1.1.2",
```

---

it's not just a bug fix, I add 2 new functions...
What do you think ?